### PR TITLE
Home page - Added some styling

### PR DESCRIPTION
**Pull request by HadiyahL**
### Proposal
Provide a short description of the change you are proposing and the reasons for it.

### Checklist

- [x] I have made this pull request to the `master` branch
- [ ] I have run `npm run lint` and there are no errors

### CODE REVIEW

**Comment by g1st** [comment]
Thanks, love the underline! It is possible to make it visible always when specific page is opened (based on pathname prop in location from react-router-dom) but not sure if we want this or not.

I am not a big fan of full screen image without any text though. Let's see what others think.

**Comment by HadiyahL** [comment]
> Thanks, love the underline! It is possible to make it visible always when specific page is opened (based on pathname prop in location from react-router-dom) but not sure if we want this or not.
> 
> I am not a big fan of full screen image without any text though. Let's see what others think.

Don't understand what you mean by  making it visible when specific page is opened.
Texts have now been added to the full screen image and the merge conflicts have been resolved.
Thanks

File: client/src/scss/index.scss [hunk 1]
```diff
@@ -11,8 +12,45 @@ main {
...
+.homepage_text h2 {
+	font-family: 'Potta One', cursive;
+	margin-bottom: 1rem;
+}
+.homepage_text li {
+	list-style-type: none;
+	display: flex;
+	align-items: center;
+	// justify-content: center;
+
```
Comment (g1st) [inline client/src/scss/index.scss:50]: Need a closing curly brace here. And you can safely remove commented out line if it is not used.

**Review by g1st — COMMENTED** [review]
Thanks its a good progress but I feel it still needs to be more polished :) In addition to the comments I have left, the text on the homepage feels like it should be of a larger font size (for desktop). And the gaps between headline, list of features and action button might be bigger also. Maybe the gaps would sort the issue of the headline starting quite low on the screen, especially on the mobile screen, where it starts basically at the middle of the screen.

Also not making the image of 100vh and making footer visible looks better to me. So if the `.jumbotron`'s `height: calc(100vh - 278px);` (100vh minus navbar and footer heights) then it looks more pleasing. But hardcoding those pixels here is a bit risky as if we change footer or navbar heights, then we need to update this. Also footer for narrow screens is higher so for mobile views `.jumbotron`s height should be different.

**Pull request merged**